frontend/src/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -155,14 +167,24 @@ class Multiselect extends React.PureComponent<Props, State> {
   }
 
   private onChange = (params: OnChangeParams): void => {
-    const newState = this.generateNewState(params)
-    this.setState(newState, () => this.commitWidgetValue({ fromUi: true }))
+    // Do NOT change state when max selection is set, we are selecting, and value.length >= this.maxSelections
```

Comment:
nit: I think it'd be more clear if this were written as

```javascript
if (this.maxSelections && params.type === "select" && this.state.value.length >= this.maxSelections) {
  return
}
this.setState(this.generateNewState(params), () => this.commitWidgetValue({ fromUi: true }))
```

You could then drop the comment entirely since it's identical to what's written in code